Apps/Sandcastle/gallery/development/Many Clipping Planes.html

```diff
@@ -0,0 +1,255 @@
+<!DOCTYPE html>
```

Comment:
I really like this demo and feel like it could be incorporated into the main clipping planes demo. If that's annoying to do it could be a standalone demo in the main folder.

Reply:
@mramato recommended moving it to Development since it's more for debugging and performance testing than demonstrating the feature. @pjcozzi also mentioned at one point that we might do actual clipping cylinders, at which point this demo kind of becomes a "wrong" way to achieve the effect.

> I really like this demo

Much appreciated though!

Reply:
I'm not sure about this being in `development`; I just spent awhile looking for it.  At the least, there should be a simple example showing using a lot of clipping planes.